src/autoscaler/metricsforwarder/forwarder/metron_emitter.go

```diff
@@ -0,0 +1,50 @@
+package forwarder
+
+import (
+	"code.cloudfoundry.org/app-autoscaler/src/autoscaler/helpers"
+	"code.cloudfoundry.org/app-autoscaler/src/autoscaler/metricsforwarder/config"
+	"code.cloudfoundry.org/app-autoscaler/src/autoscaler/models"
+	"code.cloudfoundry.org/go-loggregator/v9"
+	"code.cloudfoundry.org/lager/v3"
+)
+
+func NewMetronEmitter(logger lager.Logger, conf *config.Config) (MetricForwarder, error) {
+	tlsConfig, err := loggregator.NewIngressTLSConfig(
+		conf.LoggregatorConfig.TLS.CACertFile,
+		conf.LoggregatorConfig.TLS.CertFile,
+		conf.LoggregatorConfig.TLS.KeyFile,
+	)
+
+	if err != nil {
+		logger.Error("could-not-create-TLS-config", err, lager.Data{"config": conf})
+		return &MetronEmitter{}, err
+	}
+
+	client, err := loggregator.NewIngressClient(
+		tlsConfig,
+		loggregator.WithAddr(conf.LoggregatorConfig.MetronAddress),
+		loggregator.WithTag("origin", METRICS_FORWARDER_ORIGIN),
+		loggregator.WithLogger(helpers.NewLoggregatorGRPCLogger(logger.Session("metric_forwarder"))),
+	)
+	if err != nil {
+		logger.Error("could-not-create-loggregator-client", err, lager.Data{"config": conf})
+		return &MetronEmitter{}, err
+	}
+
+	return &MetronEmitter{
+		client: client,
+		logger: logger,
+	}, nil
+
+	return &MetronEmitter{}, nil
```

Comment:
Seems like a leftover 👀 

```suggestion
	return &MetronEmitter{
		client: client,
		logger: logger,
	}, nil
```